src/plugins/ui/sheetview.ts

```diff
@@ -157,8 +157,6 @@ export class SheetViewPlugin extends UIPlugin {
 
   handle(cmd: Command) {
     switch (cmd.type) {
-      case "UNDO":
-      case "REDO":
       case "DELETE_SHEET":
```

Comment:
then this case should be part of the existing switch/case

Reply:
I did a fixup addressing it :) It's merged together